x-pack/test/dataset_quality_api_integration/tests/integrations/integration_dashboards.spec.ts

```diff
@@ -43,42 +32,40 @@ export default function ApiTest({ getService }: FtrProviderContext) {
   registry.when('Integration dashboards', { config: 'basic' }, () => {
     describe('gets the installed integration dashboards', () => {
       before(async () => {
-        await Promise.all(
-          integrationPackages.map((pkg: IntegrationPackage) => installPackage({ supertest, pkg }))
-        );
+        await Promise.all(integrationPackages.map((pkg) => installPackage({ supertest, pkg })));
       });
 
       it('returns a non-empty body', async () => {
-        const resp = await callApiAs(integrationPackages[0].name);
+        const resp = await callApiAs(integrationPackages[0]);
         expect(resp.body).not.empty();
       });
 
       it('returns a list of dashboards in the correct format', async () => {
         const expectedResult = {
           dashboards: [
             {
-              id: 'postgresql-158be870-87f4-11e7-ad9c-db80de0bf8d3',
-              title: '[Logs PostgreSQL] Overview',
+              id: 'nginx-023d2930-f1a5-11e7-a9ef-93c69af7b129',
```

Comment:
I am always scare of such brittle tests where we rely on ID's which we are not generating and is generated by some other package. Hopefully this doesn't changes/breaks in the future

Reply:
Agree, I also thought about this. I was thinking on just assert that dashboards is not empty. Let's see how this evolves in time.